src/common/playlists/PlayHeaderInfo.jsx

```diff
@@ -31,7 +31,7 @@ const Tags = ({tags}) => {
       {tags.map((tag, index) => (
         <li key={index}>
           <span className="play-tag">
-            {tag}
+            {tag.tag.name}
```

Comment:
Change the variable name like `item.tag.name`

Reply:
@Angryman18 can you take care?

Reply:
yes fixed.